app.go

```diff
@@ -181,7 +148,56 @@ type Settings struct {
 	// Default: ".fiber.gz"
 	CompressedFileSuffix string `json:"compressed_file_suffix"`
 
-	// FEATURE: v1.13
+	// ProxyHeader will enable c.IP() to return the value of the given header key
+	// By default c.IP() will return the Remote IP from the TCP connection
+	// This property can be useful if you are behind a load balancer: X-Forwarded-*
+	// NOTE: headers are easily spoofed and the detected IP addresses are unreliable.
+	// Default: ""
+	ProxyHeader string `json:"proxy_header"`
+
+	// GETOnly rejects all non-GET requests if set to true.
+	// This option is useful as anti-DoS protection for servers
+	// accepting only GET requests. The request size is limited
+	// by ReadBufferSize if GETOnly is set.
+	// Server accepts all the requests by default.
+	GETOnly bool `json:"get_only"`
+
+	// ErrorHandler is executed when an error is returned from fiber.Handler.
+	//  cfg := fiber.Config{}
+	//  cfg.ErrorHandler = func(c *Ctx, err error) error {
+	//   code := StatusInternalServerError
+	//   if e, ok := err.(*Error); ok {
+	//     code = e.Code
+	//   }
+	//   c.Set(HeaderContentType, MIMETextPlainCharsetUTF8)
+	//   return c.Status(code).SendString(err.Error())
+	//  }
+	//  app := fiber.New(cfg)
```

Comment:
Are you sure it won't render?
<https://pkg.go.dev/github.com/gofiber/fiber?tab=doc#New>

```go
// New creates a new Fiber named instance.
//  app := fiber.New()
// You can pass optional configuration options by passing a Config struct:
//  app := fiber.New(fiber.Config{
//      Prefork: true,
//      ServerHeader: "Fiber",
//  })
func New(config ...Config) *App {
```

Reply:
That renders nicely because it's on a type definition, whereas this doc is on a field in a struct.
[For comparison](https://pkg.go.dev/github.com/gofiber/fiber?tab=doc#Settings)

Reply:
Ah nice catch, fixed 👍